source/develop/release-management/features/storage/backup-storage-domain.html.md

```diff
@@ -0,0 +1,248 @@
+* Multiple storage domain can be used for backup purposes only.
+* Backup storage domain will support both File storage(NFS, Gluster) as well as block storage(Fiber Channel and iSCSI) as compare to export storage domain which only has support for file storage.
+
+#### Restrictions
+
+* A data storage domain can not be configured as backup while there are running VMs with disks reside on that storage domain.
+* User can not run VMs with disks reside on a storage domain configured as backup, since running VM might manipulate the disk's data.
+* VMs with disks reside on a backup storage domain can not be previewed.
+* Live move of disks to the backup storage domain will be restricted.
+
+#### Open Issues
+
+* Preview will be restricted for VMs with disks reside on a backup storage. - We think it should be restricted since currently oVirt does not support import storage domain with previewed unregistered entities.
+* Shared disk will be restricted in the backup storage domain since those are not specified in the VM's OVF.
+* VM pool will be eligible in a backup storage domain although, the user must keep in mind that import storage domain will not preserve its pool reference after import.
+* Should the backup indication needs to be configured in the storage domain meta data? If so should we add the backup indication as part of V4 storage domain meta data.
+
+
+## Current progress
+
+DAL implementation:
+
+  Changes | Values
+  ---------|----------
+  status | open
+  gerrit link | https://gerrit.ovirt.org/#/c/77142/
+  DB table changed | storage_domain_static
+  Field name | backup
+  Type | True / False (Default would be false, not null)
+
+
+## Phases for Implementation
+
+- [x] Phase 1 (under review): Add dal layer with new field -
+  * introducing new field 'backup' in table storage_domain_static
+  * Add field changes in fixtures.xml for dao tests
+  * Test class added for dao test
+- [ ] Phase 2: Add command validations for configuring backup storage domain - [see restrictions]
+- [ ] Phase 3: Add REST command to update storage domain as backup - Introduce the ability to update the storage domain as backup through REST.
+
+
+## Future plans
+
+* Add GUI support for backup storage domain - Once backend configuration get complete and REST changes get implemented we will make this ready for user to use it through gui.
+* Convert Export storage domain to backup domain - If community agree then we will remove the export storage domain and replace it with the backup storage domain.
+
+
+## GUI Mockups
+
+ An Import storage domain (NFS, iSCSI) screen:
+![](/images/wiki/Import_with_backup_NFS.png)
+
+ An Add New Storage Domain (NFS, iSCSI) screen:
+![](/images/wiki/Add_new_domain_with_backup_NFS.png)
+
+
+## REST Requests
+
+### Import block Storage Domain
```

Comment:
if you are talking about GUI mockups and REST requests then they are basically for demonstration purpose so reader will know how this feature can be used. But I can remove them if they are not necessary.